invenio_rdm_records/services/generators.py

```diff
@@ -16,9 +16,10 @@
 
 from flask import current_app
 from flask_principal import UserNeed
-from invenio_access.permissions import authenticated_user
 from invenio_communities.generators import CommunityRoleNeed, CommunityRoles
 from invenio_communities.proxies import current_roles
+
+# from invenio_records.dictutils import dict_lookup
```

Comment:
```suggestion
```

Reply:
This is used in the commented part (permissions)